Makefile

```diff
@@ -129,29 +132,41 @@ $(foreach proj,$(VALID_PROJECTS),$(call include_proj,$(proj)))
 
 IGNORE_CLEANUP_LIBS := CMSIS FreeRTOS STM32F0xx_StdPeriph_Driver unity FatFs
 FIND_PATHS := $(addprefix -o -path $(LIB_DIR)/,$(IGNORE_CLEANUP_LIBS))
-FIND := find $(PROJ_DIR) $(LIB_DIR) \
+FIND := find $(PROJECT_DIR) $(LIBRARY_DIR) \
 			  \( $(wordlist 2,$(words $(FIND_PATHS)),$(FIND_PATHS)) \) -prune -o \
 				-iname "*.[ch]" -print
+FIND_MOD_NEW := git status --porcelain | grep -v -e "^D" -e "^ D" | grep -e ".*\.c" -e ".*\.h" | cut -c 4-
```

Comment:
Any reason you chose to do this instead of `git diff --name-only --diff-filter=ACMRT` like `test_format`?

Reply:
Actually I didn't know we have something like that already. The git diff command seems cleaner and already working so I'll switch to that. Thanks for pointing this out!